packages/apple-pay-integration/e2e/ApplePaySessionCustomerStepMockObject.ts

```diff
@@ -0,0 +1,130 @@
+const addApplePaySessionToChrome = () => {
+    class ApplePaySessionCustomerStep {
```

Comment:
```suggestion
    class ApplePaySessionCustomerStep implements ApplePaySession {
```

We should probably do this to ensure the mock implementation conforms to the interface we are mocking.

Reply:
Implemented the ApplePaySession